test/powershell/Modules/Microsoft.PowerShell.Utility/Write-Host.Tests.ps1

```diff
@@ -1,18 +1,91 @@
-﻿Describe "Write-Host DRT Unit Tests" -Tags "CI" {
+﻿Describe "Write-Host DRT Unit Tests" -Tags "Slow","Feature" {
```

Comment:
> "Write-Host DRT Unit Tests"

Can you change this description to be more specific? Other tests here seem also DRT unit tests.